scripts/dev-bundle-tool-package.js

```diff
@@ -14,8 +14,8 @@ var packageJson = {
     pacote: "https://github.com/meteor/pacote/tarball/a81b0324686e85d22c7688c47629d4009000e8b8",
     "node-gyp": "6.0.1",
     "node-pre-gyp": "0.14.0",
-    typescript: "3.8.3",
-    "meteor-babel": "7.9.0",
+    typescript: "4.0.5",
```

Comment:
also we need to update meteor-babel's TS to 4.0.5

Reply:
done